### PR TITLE
[DOC] Add bingo rules

### PR DESCRIPTION
**Pull request by csouchet**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,13 +104,18 @@ It can be :
...
 
 - Display a new grid with random numbers on each square
-- Add an interaction: When the user clicks on a square, a new jeton must appear on it.
-- Add an interaction: When the user clicks on a square and the row is completed, the color of the background of the whole row must be changed.
-- Add an interaction: When the user clicks on a square and the column is completed, the color of the background of the whole column must be changed.
+- Add an interaction: When the user clicks on a square, a new token must appear on it.
+- Add an interaction: When the user clicks on a square and the row is completed, the color of the background of the whole row must be changed. \
+  Add the interaction that you want to congratulate the user.
+- Add an interaction: When the user clicks on a square and the column is completed, the color of the background of the whole column must be changed. \
+  Add the interaction that you want to congratulate the user.
```
Comment (tbouffard) [inline README.md:111]:
```suggestion
- Add an interaction: When the user clicks on a square and the row is completed, the color of the background of the whole row must be changed.
- Add an interaction: When the user clicks on a square and the column is completed, the color of the background of the whole column must be changed.
- When the user wins: implement whatever you want to let him/her know he has just won (congratulations are welcome).
```

File: README.md [hunk 2]
```diff
@@ -86,13 +104,18 @@ It can be :
...
   <p align="center">
     <img title="Column" src="images/column.png">
   </p>
-- Add an interaction: When the user clicks on a square and the whole grid is completed, add the interaction that you want to congratulate the user.
+  
+**Bonus**:
+- Add an interaction: When the user clicks on a square and the diagonal is completed, the color of the background of the whole diagonal must be changed. \
+  Add the interaction that you want to congratulate the user.
```
Comment (tbouffard) [inline README.md:118]:
```suggestion
- Add an interaction: When the user clicks on a square and the diagonal is completed, the color of the background of the whole diagonal must be changed.
```

File: README.md [hunk 3]
```diff
@@ -7,7 +7,23 @@ This challenge enables the Process Analytics team working at Bonitasoft to find
 
 You work on a new Open Source project as a frontend developer. This project is destined to the Bingo players.
 
-Your mission, if you accept it, is to create a new library and a demonstrator of this library to permit the Bingo players to display a new grid with random numbers, add a new token on a square, and change the background & add an interaction when a row/column is completed.
+Your mission, if you accept it, is to create a new library, and a demonstrator of this library to permit the Bingo players to display a new grid with random numbers, add a new token on a square, and change the background & add an interaction when a row/column is completed.
+
+## Bingo rules
+### Presentation of the game
+Bingo is a game of chance played with 90 balls, numbered from 1 to 90, where the drawn numbers are announced one after the other.  \
```
Comment (tbouffard) [inline README.md:14]: ❓ If you took the following or inspired from a site, could you add a link to it for more details? Otherwise, this is fine like this.

Comment (csouchet) [reply README.md:14]: There is different rules. I mixed them to simplify the implementation.
I started from the American rules, but in thoses rules, the numbers go to 75 and the centered square is a Star.

**Review by tbouffard — APPROVED** [review]

**Pull request merged**